docs/getting_started/audioclips.rst

```diff
@@ -5,10 +5,10 @@ Audio in MoviePy
 
 This section shows how to use MoviePy to create and edit audio clips.
 
-Note that when you cut, mix or concatenate video clips in MoviePy the audio is automatically handled and you need to worry about it. This section is of interest if you just want to edit audiofiles or you want custom audio clips for your videos.
+Note that when you cut, mix or concatenate video clips in MoviePy the audio is automatically handled and you need to worry about it. This section is of interest if you just want to edit audio files or you want custom audio clips for your videos.
 
-What audioclips are made of
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+What audio clips are made of
```

Comment:
The documentation is somewhat inconsistent when it comes to spelling, but I'd opt for sticking with the spelling of MoviePy-specific class names in cases like this.

I.e. would change audioclips to AudioClips here instead of correcting to audio clips. But agree that audiofiles should get corrected to audio files.